src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -278,6 +280,18 @@
      */
     public static final Key<Value<Boolean>> ARMOR_STAND_MARKER = DummyObjectProvider.createExtendedFor(Key.class,"ARMOR_STAND_MARKER");
 
+	/**
+	 * Represents the {@link Key} for whether players are prevented from taking
+	 * items from an equipment slot on an {@link ArmorStand}
+	 */
+	public static final Key<Value<Map<EquipmentType, Boolean>>> ARMOR_STAND_TAKING_DISABLED = DummyObjectProvider.createExtendedFor(Key.class, "ARMOR_STAND_TAKING_DISABLED");
```

Comment:
We have a `Map` based `Value` subtype: [`MapValue`](https://github.com/SpongePowered/SpongeAPI/blob/1e966facc18ff0c5e9456b4ec0e9bf70f386633d/src/main/java/org/spongepowered/api/data/value/mutable/MapValue.java#L45), it's recommended to use that type in the declaration.
```suggestion
+    public static final Key<MapValue<EquipmentType, Boolean>> ARMOR_STAND_TAKING_DISABLED = DummyObjectProvider.createExtendedFor(Key.class, "ARMOR_STAND_TAKING_DISABLED");
```

Reply:
Or, alternatively, a `CollectionValue` - the `EquipmentType` simply being in the collection is enough, no need for a map-based solution here.